backend/src/schema/resolvers/notifications.spec.js

```diff
@@ -145,80 +233,188 @@ describe('given some notifications', () => {
+                  },
+                  read: false,
+                  createdAt: '2019-08-31T17:33:48.651Z',
+                }),
+                {
+                  createdAt: '2020-01-15T16:33:48.651Z',
+                  read: false,
+                  reason: 'filed_report_on_resource',
+                  from: {
+                    __typename: 'Report',
+                    id: 'reportOnUser',
+                    filed: [
+                      {
+                        reasonCategory: 'discrimination_etc',
+                        reasonDescription: 'This user is harassing me with bigoted remarks!',
+                        reportedResource: {
+                          __typename: 'User',
+                          id: 'badWomen',
+                          name: 'Mrs. Badwomen',
+                        },
+                      },
+                    ],
+                  },
+                },
+                {
+                  createdAt: '2020-01-16T10:33:48.651Z',
+                  read: true,
+                  reason: 'filed_report_on_resource',
+                  from: {
+                    __typename: 'Report',
+                    id: 'reportOnPost',
+                    filed: [
+                      {
+                        reasonCategory: 'other',
+                        reasonDescription:
+                          "This shouldn't be shown to anybody else! It's my private thing!",
+                        reportedResource: {
+                          __typename: 'Post',
+                          id: 'p4',
+                          title: 'Bad Post',
+                          content: 'I am bad content !!!',
+                        },
+                      },
+                    ],
+                  },
+                },
+                {
+                  createdAt: '2020-01-14T12:33:48.651Z',
+                  read: false,
+                  reason: 'filed_report_on_resource',
+                  from: {
+                    __typename: 'Report',
+                    id: 'reportOnComment',
+                    filed: [
+                      {
+                        reasonCategory: 'discrimination_etc',
+                        reasonDescription: 'This user is harassing me!',
+                        reportedResource: {
+                          __typename: 'Comment',
+                          id: 'c4',
+                          content: 'I am bad content in a bad comment to a bad post !!!',
+                        },
+                      },
+                    ],
+                  },
+                },
+              ]),
             },
             errors: undefined,
-          })
+          }
+
+          const response = await query({ query: notificationQuery, variables })
+          await expect(response).toMatchObject(expected)
+          await expect(response.data.notifications.length).toEqual(7) // has to be checked, because of 'arrayContaining'
```

Comment:
```suggestion
          await expect(notifications).toHaveLength(7) // has to be checked, because of 'arrayContaining'
```
what does this comment mean??

Reply:
It means that `arrayContaining` has the lack of checking the length of the array.
This is not obvious I think. Was not to me. Therefore I commented.